rest/route/host_agent.go

```diff
@@ -325,6 +325,8 @@ func (h *hostAgentNextTask) prepareHostForAgentExit(ctx context.Context, params
 // fixIntentHostRunningAgent handles an exceptional case in which an ephemeral
 // host is still believed to be an intent host but somehow the agent is running
 // on an EC2 instance associated with that intent host.
+// TODO (DEVPROD-6752): remove this once hosts have all transitioned to using
```

Comment:
Can we either move the comments over now, or make a note in DEVPROD-6752 to make sure we move them over once we delete these functions?

Reply:
The comments belong here because these are the old functions (and doc comments) that need to be deleted.